scripts/Test-Functional.ps1

```diff
@@ -0,0 +1,62 @@
+# ex: .\scripts\Test-Functional.ps1 -Action Bench -Count 2 -BenchTime "2x"
```

Comment:
Is this `count` the number of tests that can be run in parallel? If yes, then do we want to always set this to 1 so that we always get somewhat reliable timings from the benchmark tests? If several tests are running in parallel then that might affect the startup/execution time depending on the CPU, Memory used, right?

Reply:
Its the number of tests, but they dont run in parallel (I think you need to pass `-parallel` and `-cpu` flags?)
When a benchmark is run, it times the entire pass (for a total of `-benchtime`) as one unit, so you dont get individual details. `-count` allows the benchmark to be rerun, so that you can see deviation estimates